transport/workflow-client-server-secure.md

```diff
@@ -2,148 +2,148 @@
 id: secure-connection
 title: Create secure client and server
 ---
-The unity transport protocol can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
-This secure connection relies on UnityTLS and is available with the following editor versions:
-`2020.3 (2020.3.34 minimum and above),  2021.3 (2021.3.1f1 minimum and above) and 2022.1 (2022.1.0b16 minimum and above)`
+The Unity Transport package can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
+
+Secure connections are available with editor versions 2020.3 (starting at 2020.3.34), 2021.3, and 2022.1 and above.
 
 ## Server authentication
 
 :::warning Warning
-Be sure private keys are not included in your code base or client builds.
+This example uses hardcoded certificates to make understanding the process easier, but in a real deployment the server certificates should be kept separate from client builds. One way to achieve this is to put them on a separate assembly, or load them from a file on the server.
 :::
 
 ### High level authentication process
-In this configuration, the server will provide a certificate to the client (`certificate`) containing the server host name.
-The client will compare the server hostname to the one it knows (`serverName`) and will then validate the provided certificate against its own root certificate (`caCertificate`) confirming the server identity.
+
+In this configuration, the server will provide a certificate to the client (`certificate`) to prove its identity. The client will validate the certificate against its own root certificate (`caCertificate`) to validate its identity.
 
```

Comment:
> In this configuration, the server provides a certificate to the client (`certificate`) to prove its identity. The client validates the certificate against its own root certificate (`caCertificate`) to validate its identity.

I'm not sure this reads correctly. In the second sentence, it's unclear whether the client is using the certificate to validate its own or the server's identity.

Maybe this is more correct?
> In this configuration, the server provides a certificate to the client (`certificate`) to prove its identity. The client uses its own root certificate (`caCertificate`) to validate the certificate from the server.

Reply:
Yes, your suggestion is correct and much clearer. Thanks!